app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -168,6 +168,10 @@ class Config {
   String get flutterGoldChanges => 'Image changes have been found for '
       'this pull request.';
 
+  String get flutterGoldStalePR => 'This pull request has not been updated in a '
+      'while. Please update this pull request to receive results from Gold, or '
+      'close it.';
+
```

Comment:
Not sure of the wording here. Suggestions welcome. :)

Reply:
Sounds good to me but @Hixie may have comments.